app/templates/main/signed-out.html

```diff
@@ -16,8 +16,7 @@
         <div class="govuk-grid-column-two-thirds">
           <h1 class="govuk-heading-l">You have successfully signed out</h1>
           <p class="govuk-body-l">Thank you for using Access Your Records.</p>
-          <a href="#" role="button" class="govuk-button govuk-button--sign-in-again" data-module="govuk-button">
-            Sign back in
+          <a href="#" role="button" class="govuk-button govuk-button--sign-in-again" data-module="govuk-button">Sign back in
```

Comment:
Seeing as you are making changes to this bit of the codebase, Is it worth adding the href to `{{url_for("login")}}` so that the sign back in button is functional? (you will need to add `from flask import url_for` to the file if so)

Reply:
I didn't need to add the URL_for import, have updated the href now and tested @anthonyhashemi

Reply:
Or did you mean to the py file? @anthonyhashemi

Reply:
yeah sorry ignore that, forgot this is a template file not a python file 🤦